sklearn/linear_model/_bayes.py

```diff
@@ -118,6 +118,18 @@ class BayesianRidge(RegressorMixin, LinearModel):
     n_iter_ : int
         The actual number of iterations to reach the stopping criterion.
 
+    X_offset_ : array-like (length = n_samples)
+        The eventually weighted mean of regressors X, used to center X in
+        the _pre_process_data function.
```

Comment:
we shouldn't point to private methods in the public docs. In this case `_pre_process_data` is private, and in general anything starting with `_` (conventionally). Instead, you can try to find an explanation from the way these attributes are used. In this case, you could look at the `_set_intercept` to get an idea of how they work.